src/main/java/com/commercetools/helper/mapper/impl/payment/ProductNameMapper.java

```diff
@@ -37,21 +35,27 @@ public void setPrefixProductNameWithAttr(String prefixProductNameWithAttr) {
      */
     public String getPaypalItemName(@Nonnull LineItem lineItem, @Nonnull List<Locale> locales) {
         String productName = lineItem.getName().get(locales);
+        if (StringUtils.isBlank(productName)) {
+            Locale fallBackLocale = lineItem.getName().getLocales()
+                    .iterator().next();
+            productName = lineItem.getName().get(fallBackLocale);
+        }
```

Comment:
Can't we reuse it instead of having same logic in 2 classes?

Reply:
00b063221be5c47c9909e907517a4c462ddf4167